### ISSUE TITLE
Adding card button doesn't do anything

### ISSUE THREAD

**Issue by BlackTurtle123** [ollo69/ha-samsungtv-smart#89]
When clicking add to lovelace nothing is happening.
I would expect it to be added to lovelace

![image](https://user-images.githubusercontent.com/16037736/125175543-35144c00-e1cd-11eb-99b2-86b0c34a2a3f.png)

Latest updates from everything within HASSIO.
Running on rpi 4
No usefull logs can be found

I don't see in the outputs anything else besides tv and hdmi source

**Comment by dekesone** [comment]
Same issue here - can not add Samsung TV-Smart integration devices to Lovelace. Nothing found in logs.  Selecting a view from the "add to Lovelace" dialog immediately closes the window and nothing is added to the dashboard view.

**Comment by ollo69** [comment]
This doesn't seems an integration issue, more related to front-end. Did you try to add it manually to a view?

**Comment by dekesone** [comment]
Yes I've tried adding it manually.   ADD CARD - BY Entity, select Samsung tv, click Continue and the dialog disappears without asking to select which view to add it to. Nothing gets added to any views.

**Comment by ollo69** [comment]
Definitely this is not an integration issue. You could have a problem with your HA installation, with your browser or is a bug in HA front-end. Adding card wizard is not controlled by integration.
Please close this issue and eventually move to HA front-end.
